src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py

```diff
@@ -44,15 +44,15 @@
     examples:
       - name: Create/Update Managed Network
         text: |-
-               az managed-network mn create --location "eastus" --properties "{\\"scope\\":{\\"managementGroups\\":[{\\\
-"id\\":\\"/providers/Microsoft.Management/managementGroups/20000000-0001-0000-0000-000000000000\\"},{\\"id\\":\\"/provi\
-ders/Microsoft.Management/managementGroups/20000000-0002-0000-0000-000000000000\\"}],\\"subnets\\":[{\\"id\\":\\"/subsc\
-riptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/VnetC/subnets/subnetA\
-\\"},{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetwor\
-ks/VnetC/subnets/subnetB\\"}],\\"subscriptions\\":[{\\"id\\":\\"subscriptionA\\"},{\\"id\\":\\"subscriptionB\\"}],\\"vi\
-rtualNetworks\\":[{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.Network/\
-virtualNetworks/VnetA\\"},{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.\
-Network/virtualNetworks/VnetB\\"}]}}" --managed-network-name "myManagedNetwork" --resource-group "myResourceGroup"

```

Comment:
why the "scope" is missing in the changed line?

Reply:
the "scope" is flattened in codemodel, this is a fix for similar issue described in #279 .

And I checked the generated Python SDK, no need to have "scope" for ManagedNetworkProperties

![image](https://user-images.githubusercontent.com/59815250/79551203-ee1fcf80-80cb-11ea-82a1-7d2574a1a7f7.png)